berkdb/rep/rep_record.c

```diff
@@ -3764,6 +3766,10 @@ __rep_apply(dbenv, rp, rec, ret_lsnp, commit_gen, decoupled)
 	int rc, now;
 	bbtime_t start = {0}, end = {0};
 
+    if (gbl_debug_replicant_latency && (time(NULL) % 4)) {
```

Comment:
do you want time(null) % 4 == 0 (or 1) here?

Reply:
oh, I see you want this to fail more often than not.

Reply:
No this is right .. I want it to return immediately 3 out of every 4 seconds .. I've tried less than this, and couldn't make the issue repeat as reliably.